ner_v2/detectors/numeral/number/README.md

```diff
@@ -134,8 +134,11 @@ class NumberDetector(BaseNumberDetector):
     data_directory_path = os.path.join((os.path.dirname(os.path.abspath(__file__)).rstrip(os.sep)),
                                        LANGUAGE_DATA_DIRECTORY)
 
-    def __init__(self, entity_name):
-        super(NumberDetector, self).__init__(entity_name=entity_name,                                      data_directory_path=NumberDetector.data_directory_path)
+    def __init__(self, entity_name, unit_type=None):
+        super(NumberDetector, self).__init__(entity_name=entity_name,
+                                             unit_type=unit_type,
```

Comment:
Move unit_type to the end, after `data_directory_path` as `data_directory_path` is a required argument